Packages/MIES/MIES_Epochs.ipf

```diff
@@ -1015,6 +1015,7 @@ static Function [variable err, variable stimsetEndIndex] EP_AddEpochsFromStimSet
 				WaveTransform/O flip, tiInflectionPoints
 			else
 				tiInflectionPoints[] = (ec.dwStimsetBegin + IndexAfterDecimation(ec.wbOodDAQOffset + wbStimsetEpochOffset[epochNr] + wbInflectionPoints[p] + 1, ec.decimationFactor) + 1) * ec.samplingInterval
+				tiInflectionPoints[] = limit(tiInflectionPoints[p], 0, epochEnd)
```

Comment:
Just a detail, butI think it would be more correct to write
`tiInflectionPoints[] = limit(tiInflectionPoints[p], -Inf, epochEnd)`
because there is no argument for a finite lower boundary to limit here.